core/src/main/java/com/netease/arctic/op/ForcedDeleteFiles.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.op;
+
+import org.apache.iceberg.DataFile;
+import org.apache.iceberg.DeleteFile;
+import org.apache.iceberg.HasTableOperations;
+import org.apache.iceberg.PublicMergingSnapshotProducer;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.TableOperations;
+
+import static org.apache.iceberg.DataOperations.DELETE;
+
+/**
+ * Can not confirm the consistency of data, just forced delete {@link DataFile} and {@link DeleteFile}.
+ */
+public class ForcedDeleteFiles extends PublicMergingSnapshotProducer<ForcedDeleteFiles> {
```

Comment:
AFAIK, Iceberg `DeleteFiles` still can delete file even if file is already not excisted.
So do we really need this implementation?

Reply:
We need this class to delete file forced when file and position-delete are related